report_thesis/src/sections/baseline.tex

```diff
@@ -17,12 +18,24 @@
 Finally, our original replica used a random train/test split for training, in contrast to the original model's manual curation to ensure representation of extreme compositions in both sets.
 This difference stemmed from the original authors' application of domain expertize in their dataset curation --- a process we could not directly replicate.
 Nevertheless, we found that automatically identifying extreme compositions and ensuring that they were present in both the training and testing sets brought us closer to the original model.
+We chose to pull out the $n$ largest and smallest samples by concentration range, for each oxide, and reserve them for the training set.
+Then we would do a random split on the remaining dataset, such that the final train/test split would be a $80\%/20\%$ split.
 
-With these changes, we have created a more accurate replica of the \gls{moc} model, which we will use as our baseline for the rest of the paper.
-As an additional measure, we have presented these changes to one of the original authors, who confirmed that they were reasonable and in line with the original model's implementation.
+With these changes, we have created a more accurate replica of the \gls{moc} model, which we will use as our baseline for the rest of this paper.
+As an additional measure, we have presented these changes to one of the original authors of \citet{cleggRecalibrationMarsScience2017}, who confirmed that they were reasonable and in line with the original model's implementation.
 
 Table \ref{tab:replica_results_rmses} shows the \gls{rmse}s of the original models and our replicas after these changes.
 Figure~\ref{fig:rmse_histograms} illustrates the distribution of these \gls{rmse}s as a grouped histogram.
+The results show that the \gls{rmse}s of our replicas exhibit similar tendencies to the original models.
+However, in some cases, our replicas have a lower \gls{rmse} than the original models, and in others, they have a higher \gls{rmse}.
+These differences are due to a number of factors.
+
+Firstly, the original models were trained with datasets from 1600mm and 3000mm standoff distances, while we only had access to the 1600mm dataset for our replicas.
+Additionally, we automated the outlier removal for the PLS1-SM phase, unlike the original manual process.
+As mentioned, the original authors manually curated their training and test sets, ensuring a broad elemental range, while implemented an automatic process for our replicas due to lack of domain expertise.
+Differences might also stem from varied implementation specifics, such as programming languages and libraries used.
+Lastly, \gls{rmse} is simply a statistical measure of the differences between the actual and predicted values
+It does not necessarily describe the actual model accuracy on new data from Mars, so the results should be interpreted with this in mind.
```

Comment:
```suggestion
Lastly, \gls{rmse} is simply a measure of the differences between the actual and predicted values
It does not necessarily describe the actual model accuracy on new data from Mars, so the results should be interpreted with this in mind.
```
You could write this, but just remove the `statistical`, as above.
Not sure if I like this part, though. It's essentially:
- We proved in our previous work that we've made a pretty accurate replica
- We further improved it this time, taking what we learned and implementing that
- But there remain a few differences for X, Y, Z reasons
- But that doesn't actually matter much because what matters is how the models perform on the actual data from Mars

The last point doesn't matter much here IMO. We don't have a way to check whether the models generalize to Mars data. And it seems kind of obvious, no?

Reply:
We could just remove it then. It was just an argument we had in the previous paper, that's why I brought it back up here.

I've removed it in a commit.